src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,6 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty lines before the class declaration.